files/generate_django_secret_key.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
are you sure this is needed? The virtualenv should know the python version without the need of introducing this change

Reply:
Not sure now, but I probably made this change and then forgot to remove it :grin: